src/repr/src/scalar.rs

```diff
@@ -1135,6 +1135,13 @@ impl fmt::Display for Datum<'_> {
             }
             Datum::Uuid(u) => write!(f, "{}", u),
             Datum::Array(array) => {
+                if array.dims().into_iter().any(|dim| dim.lower_bound != 1) {
```

Comment:
I think my platform check will cover this, adding it now.

Reply:
@def- I hadn't realized you pushed some tests to this branch and inadvertently blew them away. Would you mind pushing them back? I need to figure out some better workflow to prevent this from happening so often; many apologies.

Reply:
Done, no worries. I'd have similar problems as I enjoy rebasing and force-pushing on my PRs too.